### PR TITLE
Updated changelog

### PR DESCRIPTION
**Pull request by cdce8p**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ Sections
 ### Developers
 -->
 
+## [2.2.3] - 2018-10-05
+
+### Added
+- Added `NeoPixelsLightStrip` accessory. [#144](https://github.com/ikalchev/HAP-python/pull/144)
+- Added new Accessory categories. [Commit](https://github.com/ikalchev/HAP-python/commit/dbf1d5d8fea814af52098f3a2f3cd5a47cd889c1)
```
Comment (cdce8p) [inline CHANGELOG.md]: If you first push the new accessory categories, we could update this PR and then merge it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,20 @@ Sections
 ### Developers
 -->
 
+## [2.2.3] - 2018-10-05
```
Comment (cdce8p) [inline CHANGELOG.md]: I assumed you want to go with `2.2.3`. If not we can change that.

**Comment by ikalchev** [comment]
It's all good - I will add the categories and also the camera (I don't know if you re following the discussion on that but I think it is ready at least as a beta). But I can merge this add them myself :)

**Comment by cdce8p** [comment]
Yeah, at least partially. As long as it doesn't break anything else, why not.

**Pull request merged**